app/src/main/java/com/example/lhi06/residue/Album.java

```diff
@@ -2,11 +2,18 @@
 
 import android.support.annotation.NonNull;
 
+import com.google.gson.annotations.SerializedName;
+
 public class Album {
 
-    private final String collectionName;
-    private final String artistName;
-    private final String artworkUrl100;
+    @SerializedName("collectionName")
```

Comment:
If the field name is the same as the JSON attribute then there is no need to have `@SerializedName` as this only tells GSON how to map the fields in case you don't want the same names in your POJO